app/src/main/java/org/jellyfin/androidtv/ui/startup/AddServerAlertFragment.kt

```diff
@@ -42,9 +42,14 @@ class AddServerAlertFragment : AlertFragment() {
 		with(binding.address) {
 			onFocusChangeListener = KeyboardFocusChangeListener()
 			nextFocusForwardId = parentBinding.confirm.id
+			nextFocusDownId = parentBinding.confirm.id
 			imeOptions = EditorInfo.IME_ACTION_DONE
 			setOnEditorActionListener { _, actionId, _ ->
-				if (actionId == EditorInfo.IME_ACTION_DONE) parentBinding.confirm.performClick()
+				if (actionId == EditorInfo.IME_ACTION_DONE) {
+					clearFocus()
+					parentBinding.confirm.performClick()
+					return@setOnEditorActionListener true
```

Comment:
The return statement is unnecessary, we return the returned value of the `performClick()` (also a boolean)

```suggestion
```

Reply:
The performClick() function always returns true because it contains an eventlistener. But returning it explicitly might be easier to understand. In that case you should use:

```suggestion
					true
```

Because the last statement of this function is the returned value

Reply:
Fixed now... I'm still not used to Kotlin's implicit return stuff. :smile: